shared/chat/avatars.js

```diff
@@ -31,9 +31,9 @@ const MutedIcon = ({isHovered, isMuted, isSelected, isLocked}) => {
   let type
   if (isMuted) {
     if (isMobile) {
-      type = isSelected ? 'icon-shh-active-24' : 'icon-shh-24'
+      type = isSelected ? 'icon-shh-active-26-21' : 'icon-shh-26-21'
```

Comment:
why do these have this naming convention

Reply:
It's their dimensions. They're not square.